net/tcpreplay/Makefile

```diff
@@ -127,6 +127,10 @@ endef
 CONFIGURE_ARGS += \
 	--enable-force-pf \
 	--enable-dynamic-link \
+	ac_cv_lib_nl_nl_cache_alloc=no \
+	ac_cv_lib_nl_genl_3_genl_connect=no \
+	ac_cv_lib_nl_3_nl_cache_alloc=no \
+	ac_cv_lib_dbus_1_dbus_malloc=no \
 	--prefix="$(PKG_INSTALL_DIR)/usr" \
```

Comment:
It looks like you have mixed CONFIGURE_ARGS and CONFIGURE_VARS

That might work, but in the long run it is probably best to keep them separately.